### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by dblack850** [FrenzyRoleplay/server-issues#31]
**Describe the bug**
After shooting a gun in first-person from a police vehicle, the windshield turns black/opaque.

**To Reproduce**
1. Drive a law enforcement Dodge Charger (possibly other vehicles, I will try to check when I'm next in-city).
2. Aim and shoot out of the front windshield (possibly other vehicles, I will try to check when I'm next in-city).

**Expected behavior**
Bullet holes, broken glass, or some way in which there would still be visibility.

**Screenshots**
Please watch for context and good lulz:
https://clips.twitch.tv/SeductiveThankfulPorcupineDoritosChip

**Additional context**
The recent change to shooting from vehicles is great for immersion, balance purposes, etc. I love it. However, as an LEO, this issue with the Charger makes it difficult to shoot from a vehicle at all if not something to avoid altogether. While criminals could have passengers shooting out of the back of a transparent vehicle, possibly automatic gunfire, LEO with this issue with the Charger windshield would be driving and shooting at the same time, in first-person, only to have their windshield go black and make it 'impossible' to shoot or cause a crash which ends a chase/gun-fight prematurely (as imaginable from the linked clip). I'm not sure if the Charger is fixable but, if this is a consistent issue, it would cause the lack of the ability to shoot and drives as an LEO in most departments' primarily-used vehicle.

**Comment by badcodestv** [comment]
- Are you able to replicate this more than once?
- Are you running any visual mods?
- What are your graphic settings at?

**Comment by dblack850** [comment]
> * Are you able to replicate this more than once?
Yes, I can confirm. I just logged into the city on the live version and withdrew an LSPD Dodge Charger to do a little test.

> * Are you running any visual mods?
In the video, I was using a visual mod (LP) but, after seeing your question and realizing that might have something to do with it, I uninstalled all graphics mods for my test this morning so that I could see if I could replicate the bug without visual mods.

> * What are your graphic settings at?
I'm hoping my problem might be here. Here are screenshots in an album (in the last screenshot, you can see the tint overlay on the front windshield of the LSPD Dodge Charger):
https://imgur.com/a/Ih7eayZ

Here is a video. without mods, showing the other windows of the LEO Dodge Charger's windows tinting/turning black/turning opaque after being shot. I can't attach it because it's 12MB and the limit is 10 so I've uploaded it to YouTube. Note: There were only 7 people online when I did this and it was done at the Ammunation parking lot in Sandy Shores just in case.

https://www.youtube.com/watch?v=4tqd9IZfn4k

**Comment by dblack850** [comment]
This problem doesn't exist with the Ford Taurus or Chevrolet Impala, though you can't turn to aim out of the back windshield (assuming there's a "cage"/barrier that's supposed to be there or enabled via extras).

This problem doesn't exist at all for the Ford Crown Victoria.
